qpicard/src/org/qcmg/picard/util/BAMFileUtils.java

```diff
@@ -46,4 +52,47 @@ public static List<String> getContigsFromHeader(SAMFileHeader header) {
 				.collect(Collectors.toList());
 	}
 
+	/**
+	 * picard index name created by replacing .bam to .bai. But we want to have index name end with .bam.bai
+	 * @param bamFile
+	 * @throws Exception 
+	 */	
+	public static void renameBamIndex(File output) throws IOException{
```

Comment:
It's usually a good idea to check the input for `public static` methods. What will happen here if output is null?

Reply:
thanks. I will check the input.